Dockerfile

```diff
@@ -1,12 +1,14 @@
-FROM node:8
+FROM node:12.17
 
 RUN mkdir /twilio
 WORKDIR /twilio
 
 ENV NODE_PATH /usr/local/lib/node_modules
 
 COPY lib ./lib
+COPY spec ./spec
 COPY index.js .
 COPY package* ./
 
 RUN npm install . -g
+RUN npm install jasmine -g
```

Comment:
Why does jasmine not get installed as part of the package install?

Reply:
Dev dependencies seem to get ignored when you run a global npm install; think installing normally and adding a `--include=dev` flag makes more sense since it's unclear what we're gaining from doing a global install.